package.json

```diff
@@ -46,12 +46,13 @@
     ],
     "testPathIgnorePatterns": [
       "/node_modules/",
+      "<rootDir>/integration-tests/create-react-app",
       "<rootDir>/packages/react-percy-webpack/src/entry/__tests__/test-project/"
     ]
   },
   "lint-staged": {
     "*.js": [
-      "eslint --fix --color --no-ignore",
```

Comment:
We definitely want local linting on lint-staged.  If it's hard to make lint-staged behave, then renaming `.storybook` to `storybook` sounds like a good option.  I guess I'm not clear on why Storybook prepends this with the . in the first place, considering you always need to edit files inside it.